docker-compose.dev.yaml

```diff
@@ -31,16 +31,19 @@ services:
       - ipfs_data:/data/ipfs
 
   content-publishing-service-api:
+    pull_policy: never
+    image: content-publishing-service
     build:
       context: .
       dockerfile: dev.Dockerfile
+      tags:
+        - content-publishing-service:latest
     ports:
       - 3000:3000
     env_file:
       - .env.docker.dev
     environment:
       - START_PROCESS=api
-      - REDIS_URL=redis://redis:6379
```

Comment:
Question: I'm curious how we are injecting the redis url or maybe that was redundant?